sig/responses/add_api_key_response.rbs

```diff
@@ -0,0 +1,9 @@
+module Algolia
+  class AddApiKeyResponse < BaseResponse
+    attr_reader raw_response: Hash[Symbol, String|[String]]
+
+    # TODO add client
```

Comment:
I think we can address all the `# TODO`s in this PR, right? I believe we'll have (almost) all types ready now.

Reply:
I'll address them in a different PR! This only mainly serves as putting back the lost commits 😢